internal/platform/blockchain/ERC721BridgelessMinting/ERC721BridgelessMinting.go

```diff
@@ -0,0 +1,1282 @@
+// Code generated - DO NOT EDIT.
```

Comment:
maybe this file we should placed in a folder `autogenerated`? And put readme how it is autogenerated. just 2 3 lines readme. abigen command. Just a suggestion

Reply:
We decided to structure it this way:
```
internal
  platform
    blockchain
      contract
        erc721universal.go
```
Does that work for you?

We have not written any instructions as for now there is no need for more contracts

Reply:
yes, the folder name `contract` is OK. I like it.